api/setup.py

```diff
@@ -60,6 +59,7 @@ def recur_expand(target_root, dir):
     entry_points={
         'console_scripts': [
             'zulip-send=zulip.send:main',
+            'zulip-bot-server=zulip.zulip_bot_server:main',
         ],
```

Comment:
Another recommendation, in `console_scripts`, if the command is already named `zulip-bot-server`, then the file should just be named `bot_server.py`. So let's rename it to `bot_server.py`, the above entry then would look like `'zulip-bot-server=zulip.bot_server:main'`. :)